packages/payload/src/bin/generateImportMap/utilities/resolveImportMapFilePath.ts

```diff
@@ -17,17 +17,29 @@ export function resolveImportMapFilePath({
 
   if (importMapFile?.length) {
     if (!fs.existsSync(importMapFile)) {
-      throw new Error(`Could not find the import map file at ${importMapFile}`)
+      try {
```

Comment:
One thing while you're at it - could you make this function async and use async fs calls?